### PR TITLE
Test for empty comments to fix regression in #33

### LINKED ISSUE #34: Empty comment causes exception
Running this plugin on code with comments that are just `# noqa` gives me:

```
  File "/usr/local/lib/python3.8/site-packages/flake8_spellcheck/__init__.py", line 170, in _parse_token
    and token_info.string.lstrip("#").split()[0] != "noqa:"
IndexError: list index out of range
```

### PR DESCRIPTION
**Pull request by 3n-k1**
Closes #34

thought it was a bit funny that flake8 reported your name as a typo ;)

### CODE REVIEW

**Comment by 3n-k1** [comment]
oops, forgot to hit save after removing an errant comment

File: tests/test_flake8_spellcheck.py [hunk 1]
```diff
@@ -104,6 +104,12 @@ def test_flake8_pragma_spaces(self, flake8dir):
             "./example.py:1:14: E262 inline comment should start with '# '"
         ]
 
+    # Regression test for github.com/MichaelAquilina/flake8-spellcheck/issues/34
+    def test_empty_comment(self, flake8dir):
+        flake8dir.make_example_py("#")
```
Comment (MichaelAquilina) [inline tests/test_flake8_spellcheck.py:109]: could you expand this a bit more? Maybe something like

```python
flake8dir.make_example_py("""
    # the line below should not fail
    #
    # hello world
""")
```

Comment (3n-k1) [reply tests/test_flake8_spellcheck.py:109]: sure!

File: whitelist.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+Aquilina
```
Comment (MichaelAquilina) [inline whitelist.txt:1]: :D Guess that's fair

**Comment by 3n-k1** [comment]
no problem at all! thanks for merging the original pr, it's been very helpful c:

**Pull request merged**